main.tf

```diff
@@ -14,3 +14,90 @@ resource "aws_kms_alias" "default" {
   name          = coalesce(var.alias, format("alias/%v", module.this.id))
   target_key_id = join("", aws_kms_key.default.*.id)
 }
+
+locals {
+  # Set default of "all of this account" for policy groups
+  policy_key_admins   = var.policy_key_admins != null ? var.policy_key_admins : ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"]
+  policy_key_users    = var.policy_key_users != null ? var.policy_key_users : ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"]
+  policy_key_grantors = var.policy_key_grantors != null ? var.policy_key_grantors : ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"]
```

Comment:
I don't think defaulting to "entire account" is a sane default, I think it should error out and force a user of the module to specify these vars. I feel like it'll be too easy for someone to unknowingly expose their kms keys to a wider surface area than expected, because despite it being on the enduser to read a module they are consuming/using, I think airing on the side of caution can help. We know not everyone is going to read the fine print as they say.